payments_service/auth/tests/test_utils.py

```diff
@@ -0,0 +1,37 @@
+from django.core.exceptions import ObjectDoesNotExist
+
+from payments_service.base.tests import TestCase
+
+from .. utils import set_up_braintree_customer
```

Comment:
Is this common place? I rarely saw it in webdev stuff I worked on... it feels less explicit and readable than `payments_services.auth.tests.utils`... even though I get it's much shorter.

If there's a fair bit of precedent in the code base no worries, hardly that important.